templates/taxi/driver_list.html

```diff
@@ -4,10 +4,13 @@
   <h1>Driver list</h1>
```

Comment:
Make sure to use 2 whitespaces indentations in your `.html` files as per the checklist.